src/data/text/move_descriptions.h

```diff
@@ -1426,7 +1426,7 @@ static const u8 sGRAVITYDescription[] = _(
     "negating levitation.");
 
 static const u8 sMIRACLE_EYEDescription[] = _(
-    "Makes GHOSTS and evasive\n"
+    "Makes Dark-types and evasive\n"
     "foes easier to hit.");
```

Comment:
Should be "Dark-type" instead of "Dark-types" to fit the pixel width.

Could also remain "Dark-types" with "evasive" being dropped to the second line.